openhtf/io/frontend/src/app/utils.ts

```diff
@@ -41,6 +41,7 @@ export class SimplifyStatus extends MapPipe {
     'WAITING_FOR_TEST_START': 'WAITING',
     'STOP_WAIT': 'WAITING',
     'TIMEOUT': 'ERROR',
+    'ABORTED': 'ERROR',
```

Comment:
Values here are now in `openhtf.exe.test_state.TestState.Status`:
 - WAITING_FOR_TEST_START
 - RUNNING
 - COMPLETED

Not to be confused with `openhtf.io.test_record.Outcome`:
 - PASS
 - FAIL
 - ERROR
 - TIMEOUT
 - ABORTED